src/components/layout.js

```diff
@@ -25,6 +25,16 @@ const Layout = ({ children }) => {
 
   return (
     <>
+      <Helmet
```

Comment:
I think that metadata/helmet stuff should go in the Layout, above the header field. Would it be possible to take this part out of the PR and just focus on the page rendering correctly, without SEO stuff? I can work on the header/helmet stuff as part of migrating the existing homepage.